Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss

```diff
@@ -304,7 +300,7 @@ $snackbar-padding-y-multi: 1.25rem !default;
 .snackbar-stack-top-start .snackbar-hide,
 .snackbar-stack-top-end .snackbar-hide {
     transition: all var(--stack-transition-duration, 1000);
-    animation: hideTop var(--stack-animation-duration, 500) forwards;
+    animation: hideTop var(--stack-animation-duration, 500) backwards;
```

Comment:
Why is the animation changed to backwards? WAs it wrong before? Or will this now produce a different behaviour that users might not be expecting on their apps?

Reply:
Same question from me. Even if I revert it to `forwards` the animation for stacked snackbars still doent's work. When snackbar hides it should hide from bottom to top when it is placed on top of the screen. At the moment it closes from top to bottom, which seems strange.